src/AFL-3.0.xml

```diff
@@ -0,0 +1,208 @@
+<spdx name="Academic Free License v3.0" identifier="AFL-3.0" osi-approved="true">
+  <urls>
+    <url>http://www.rosenlaw.com/AFL3.0.htm</url>
```

Comment:
Not 100% sure we have a policy that we will remove currently non-working URLs -- we might want to keep them for archival purposes, and just add new URLs as new authoritative(-ish) URLs come up. 

Since this affects a number of licenses, can you open an issue where we can decide how to handle linkrot (or document it, if there's already a policy), before any more PRs?